docs/ADR-48-locking-collections-in-the-builder.md

```diff
@@ -42,7 +42,29 @@ function isLocked(element: Collection | Item) {
 }
 ```
 
-Anyone consuming this data can read this property and disable all components that allow changes that would create a desync, like changing the name of a Decentraland Collection for example.
+### What's being locked
+
+As we said before, the locking mechanism is in place so we can avoid desynching the database with the blockchain. Anyone consuming this data can read this property and disable all components that change data that's being commited to the Blockchain.
+
+- **Decentraland**: Once the collection is published, every property that changes the address should not be updated. These are:
+
+  - The name
+  - Adding or removing items
+  - Changing items data:
+    - rarity
+    - price
+    - beneficiary
+    - metadata
+      - type
+      - name
+      - description
+      - data (the BodyShape and Category)
+
+- **Third Party**: Once the items are published, the URN cannot change anymore. That means that item `token_id`s and virtual collection `urn_suffix`es. For reference, and URN looks like this: `urn:decentraland:{network}:collections-thirdparty:{third-party-name}:{collection-id}:{item-id}` where
+  - network: the network the item is in, `mumbai`, `polygon`, etc
```

Comment:
`matic` instead of `polygon` right?

Reply:
Shoot, right, I always make a mistake with [common schema's name](https://github.com/decentraland/common-schemas/blob/main/src/dapps/chain-id.ts#L63)